### PR TITLE
Add BLS multisignatures

### PR DESCRIPTION
**Pull request by fyrchik**
This PR implements interfaces for multisignatures and BLS as a default implementation.

### CODE REVIEW

File: multisig/multisig.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import (
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/pairing/bn256"
+	"go.dedis.ch/kyber/v3/sign/bls"
+	"go.dedis.ch/kyber/v3/util/random"
+)
+
+type (
+	PrivateKey = kyber.Scalar
+	PublicKey  = kyber.Point
```
Comment (im-kulikov) [inline multisig/multisig.go]: Add doc-comments for this types

File: multisig/multisig.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	return bls.NewKeyPair(suite, random.New())
+}
+
+// Sign returns message signature.
+func Sign(key PrivateKey, msg []byte) ([]byte, error) {
+	return bls.Sign(suite, key, msg)
+}
+
+// Verify checks if sig is msg signature.
+func Verify(pub PublicKey, msg, sig []byte) error {
```
Comment (im-kulikov) [inline multisig/multisig.go]: SignRFC6979 / Verify has different function signature.. mb we can change signature here, what you think?

**Review by im-kulikov — CHANGES_REQUESTED** [review]
rejected

**Comment by realloc** [comment]
Not needed now

**Pull request closed without merging**